src/react/src/components/FacilityDetailsDetail.jsx

```
@@ -66,9 +83,17 @@ const FacilityDetailsDetail = ({
         </ShowOnly>
         <div>
-            <Typography className={classes.primaryText}>
+            <Typography className={classes.primaryText} component="div">
                 {primary || locationLabeled}
+                {unit ? <span className={classes.unitText}>{unit}</span> : null}
             </Typography>
+            {sourceBy ? (
+                <Typography
+                    className={classes.sourceText}
+                    component="div"
+                    dangerouslySetInnerHTML={{ __html: sourceBy }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)